components/eventing-controller/controllers/subscriptionv2/jetstream/reconciler_integration_test.go

```diff
@@ -456,34 +505,6 @@ func TestChangeSubscription(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "resolve multiple conditions",
```

Comment:
What is the reason for removing this test? Are they covered somewhere else?

Reply:
This test is no longer valid since we cannot create a subscription without filters and from [this](https://github.com/kyma-project/kyma/issues/15706) issue description, this test was not needed.